app/pb_kits/playbook/pb_icon/_icon.html.erb

```diff
@@ -1,7 +1,4 @@
-<% if !object.icon_class.nil? %>
-  <%= content_tag(:i, nil,
-      id: object.id,
-      data: object.data,
-      class: object.classname(object.kit_class),
-      aria: object.aria) %>
```

Comment:
We want to keep `aria` in there. It's a base prop.

Reply:
Added back in